controls/radrichtextbox/features/content-controls/content-controls.md

```diff
@@ -42,7 +42,7 @@ The following content controls are not supported by __Microsoft Word__:
 
 ## Common Content Controls Properties
 
->note Some of the content controls` properties are marked as **(Non-exportable)** which means they live only in the RadRichTextBox model.
+>note Some of the content controls` properties are marked as **(Non-exportable)** which means they are specific only for RadRichTextBox and are not serialized to any of the supported document formats. These properties are included because they are considered useful and could enhance the experience using the content controls in a RadRichTextBox environment.
```

Comment:
How are they useful? Add some description next to the properties to describe that

Reply:
I think these properties are understandably described.